### DISCUSSION TITLE
Release 22 is missing

### DISCUSSION THREAD

**Discussion by nZeus** [VerifyTests/Verify#1015] [General]
Hi @SimonCropp ,

[Similar to v21](https://github.com/VerifyTests/Verify/issues/986), there is no release published for v22.
Could you please add it there?

Thank you

**Comment by SimonCropp** [comment]
it is too much keep GH releases ip to date. especially when they are only a copy of the issues in a milestone. instead have a look at the issues in the relevant milestone https://github.com/VerifyTests/Verify/milestones?state=closed

**Reply by nZeus** [reply]
Hi Simon, thank you for the explanation. I understand your point of view.
TBH I think release is more important than the relevant milestone, as such tools like renovate generate links to "Release" section, also GitHub shows the latest release on the main page. When release is missing then people get confused (like this happened with v21 and v22). While nobody cares about milestones, as they rarely show the real state.

A few days ago you released a new major version (v22), and people might be interested to know what were the breaking changes. I checked the milestone, and checked commits, but I couldn't find what were the breaking changes... Could you help me to identify the breaking changes, please?

Thank you and good luck with this nice library. We enjoy using it
